src/pages/Attributions.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Schedule() {
```

Comment:
Hey Justin! Please change the function name to match the name of the page. Other than that, good work!

Reply:
I just updated it